src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts

```diff
@@ -326,20 +324,22 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
   public addSelectedRegion(event: MatAutocompleteSelectedEvent): void {
     let id;
     let name;
+    let nameUa;
     const selectedItemName = event.option.value;
     const selectedItem = this.locations.filter((element) => element.regionTranslationDtos.find((it) => it.regionName === selectedItemName));
-    const regionNameUa = selectedItem[0].regionTranslationDtos.find((item) => item.languageCode === Language.UA).regionName;
-    const regionNameEn = selectedItem[0].regionTranslationDtos.find((item) => item.languageCode === Language.EN).regionName;
-    this.selectedRegionValue.push({ regionNameUa, regionNameEn });
 
     selectedItem.forEach((item) => {
       id = item.regionId;
       name = item.regionTranslationDtos
-        .filter((it) => it.languageCode === this.currentLanguage)
+        .filter((it) => it.languageCode === Language.EN)
```

Comment:
maybe moving this duplicated code into a separate function will be better?

Reply:
Thank you! The separate mrthod was created.